Style changes

jnewbery opened the pull request:
- Centers titles and date
- adds date to newsletter posts
- adds space between the horizontal bar and text

blog post before:

![image](https://user-images.githubusercontent.com/1063656/43208108-a403e5ec-8ff7-11e8-8fbd-5dba238390bb.png)

blog post after:

![image](https://user-images.githubusercontent.com/1063656/43208132-b58d2300-8ff7-11e8-8db5-af4b79cc0bce.png)

newsletter before:

![image](https://user-images.githubusercontent.com/1063656/43208169-ce2e0cd0-8ff7-11e8-8701-58dfccb1f338.png)

newsletter after:

![image](https://user-images.githubusercontent.com/1063656/43208184-d812882a-8ff7-11e8-814f-081b6a965cad.png)

jnewbery commented:
@jamesob has already visually reviewed these changes

harding commented:
utACK.

This doesn't add the bar and it does make it look better, so technically this paragraph is out of scope for this PR, but I'll note that I don't understand why the bar is there and think the page would look better without it (but it's not a big deal for me).

jnewbery commented:
> I don't understand why the bar is there and think the page would look better without it 

Agree - I've removed the bar (I think that might have been added by a copy-paste error from the front-matter :flushed: )

I also noticed that this branch centered the `h1`s in the about, newsletter and workshop pages, which I don't think we want. I've updated the commit to add a new post.css file.
